src/lib/components/EndpointList.tsx

```
@@ -11,20 +11,24 @@ const EndpointItem: FC<{ method?: HTTP_METHOD; url?: string }> = ({
   </div>
 );
 
 const EndpointList: FC<{
   endpoints?: EndpointItem[];
   method?: HTTP_METHOD;
   url?: string;
-}> = ({ endpoints, url, method }) => {
+  title?: string;
+  description?: string;
+}> = ({ endpoints, url, method, title, description }) => {
   const listMode = endpoints && endpoints.length;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)